.azure/pipelines/jobs/default-build.yml

```diff
@@ -134,11 +124,8 @@ jobs:
         Write-Host "##vso[task.prependpath]$env:JAVA_HOME\bin"
 
       displayName: Install JDK 11
-    - powershell: |
-        Write-Host "##vso[task.setvariable variable=SeleniumProcessTrackingFolder]$(BuildDirectory)\obj\selenium\"
-        ./eng/scripts/InstallGoogleChrome.ps1
-
-      displayName: Install Chrome
+    - powershell: Write-Host "##vso[task.setvariable variable=SeleniumProcessTrackingFolder]$(BuildDirectory)\obj\selenium\"
+      displayName: Set selenium process tracking folder
```

Comment:
To confirm: Chrome will be installed by default? Cool&hellip;

Reply:
Should be after https://dnceng.visualstudio.com/internal/_git/dotnet-helix-machines/pullrequest/971?_a=overview is deployed